controllers/resources_controller.py

```diff
@@ -116,6 +118,25 @@ def index(self):
             search_text, active_resource_type, session
         )
 
+        # get config generator URL
+        config_generator_service_url = self.handler().config().get(
+            "config_generator_service_url",
+            "http://qwc-config-service:9090"
+        )
+        url = urljoin(config_generator_service_url, "resources")
+        tenant = self.handler().tenant
+        response = requests.get(url, params={'tenant': tenant})
+        if response.status_code != requests.codes.ok:
+            self.logger.error(
+                "Could not get all resources from %s:\n%s" %
+                (response.url, response.content)
+            )
+
+        # List of resources that are referenced somewhere in the config of a
+        # service
+        resources_from_config = response.json()
```

Comment:
This will likely fail here or later on line 183 if the above request failed. `resources_from_config` should always be a list (maybe empty).